android/quest/src/main/java/org/smartregister/fhircore/quest/util/PatientUtil.kt

```diff
@@ -52,7 +54,12 @@ suspend fun loadAdditionalData(
       val conditions =
         getSearchResults<Condition>("Patient/$patientId", Condition.SUBJECT, filter, fhirEngine)
 
-      val sortedByDescending = conditions.maxByOrNull { it.recordedDate }
+      val sortedByDescending =
+        conditions.maxByOrNull {
+          it.meta.lastUpdated?.asYyyyMmDdHHmmss()
+            ?: it.recordedDate?.asYyyyMmDd()?.plus(" 23:60:60") ?: ""
```

Comment:
can you add a comment explaining why we add that specific time

Reply:
This actually pulled mistakenly from other branch that was added by Maimoona and reverted later on. I am cleaning up my PR and re-submit after removing such code.